misk-aws/src/main/kotlin/misk/jobqueue/sqs/SqsJobConsumer.kt

```diff
@@ -45,16 +55,13 @@ internal class SqsJobConsumer @Inject internal constructor(
     }
 
     log.info {
-      "subscribing to queue ${queueName.value} with ${config.concurrent_receivers_per_queue} receivers"
+      "subscribing to queue ${queueName.value}"
     }
-
-    for (i in (0 until config.concurrent_receivers_per_queue)) {
-      taskQueue.scheduleWithBackoff(Duration.ZERO) {
-        // Don't call handlers until all services are ready, otherwise handlers will crash because
-        // the services they might need (databases, etc.) won't be ready.
-        serviceManagerProvider.get().awaitHealthy()
-        receiver.runOnce()
-      }
+    taskQueue.scheduleWithBackoff(Duration.ZERO) {
+      // Don't call handlers until all services are ready, otherwise handlers will crash because
+      // the services they might need (databases, etc.) won't be ready.
+      serviceManagerProvider.get().awaitHealthy()
+      receiver.runOnce()
```

Comment:
`runOnce` is misleading. This method will be run multiple times. what about `receive` or `consume`?